.travis.yml

```diff
@@ -38,3 +38,4 @@ env:
   - IDEA_VERSION=3.6
   - IDEA_VERSION=2019.2.4
   - IDEA_VERSION=2019.3
+  - IDEA_VERSION=4.0
```

Comment:
Just confirming that you do intend to compile against 4.0 as part of this PR.

Reply:
Not really. That's the dev channel build, which isn't (currently) built on travis. It does go through setup but the plugin tool exits without doing any work. Perhaps the `plugin generate` function should be changed to only generate the builds for the given channel; I haven't gotten so far into this that I know how that will work yet.